skyportal/tests/conftest.py

```diff
@@ -1638,3 +1645,48 @@ def analysis_token(user):
     )
     yield token_id
     delete_token(token_id)
+
+
+@pytest.fixture(scope="function")
```

Comment:
If you set `scope='session'` you would not need to re-load them for each test.

Reply:
I think we want to reload them when we also delete and add to them. Or maybe that is taken care of?

Reply:
How important is it for the tests if you make some modifications to each dataset? 
How much data is in each file? 
If there's not a lot of data and the changes made by each test are substantial, then leaving it with function scope is right. 
If you load a bunch of data and it takes 10 minutes to load and then the test makes some small changes to it that don't change the outcome of other tests, then session scope.